src/averageAges.js

```diff
@@ -37,7 +51,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const WOMAN = 'f';
```

Comment:
Create reusable constants and functions outside of the function, to keep your code DRY.